apps/src/applab/appStorage.js

```diff
@@ -5,11 +5,17 @@
  */
 var AppStorage = module.exports;
 
-// TODO(dave): remove once we can store ids for each app.
-AppStorage.tempEncryptedAppId =
+// TODO(dave): remove once all applab data levels are associated with
+// a project.
+AppStorage.tempChannelId =
     window.location.hostname.split('.')[0] === 'localhost' ?
         "SmwVmYVl1V5UCCw1Ec6Dtw==" : "DvTw9X3pDcyDyil44S6qbw==";
 
+AppStorage.getChannelId = function() {
```

Comment:
Per offline discussion, the option of using appOptions is available, but creating a new channel when /p/artist or /p/applab is loaded would create a number of complications in the project code. Because this doesn't make the problem substantially worse, I'm just going to do it this way for now.

The upgrade path once the channel id is available in appOptions is straight forward. I've added a TODO here reminding myself that this needs to be done eventually.